loki/backend/cgen.py

```diff
@@ -143,48 +175,59 @@ def visit_Module(self, o, **kwargs):
         routines = self.visit(o.routines, **kwargs)
         return self.join_lines(spec, routines)
 
-    def visit_Subroutine(self, o, **kwargs):
+    def _subroutine_header(self, o, **kwargs):
         """
-        Format as:
-
-          ...imports...
-          int <name>(<args>) {
-            ...spec without imports and argument declarations...
-            ...body...
-          }
+        Helper function/header for :func:`~loki.backend.CCodegen.visit_Subroutine`.
```

Comment:
As far as I know, if you prefix the content with `~`, the link text will only be `visit_Subroutine` instead of `loki.backend.CCodegen.visit_Subroutine`.

Reply:
Ah, I didn't know. Neat!